addons/hr/models/res_partner.py

```diff
@@ -16,7 +16,13 @@ def get_static_mention_suggestions(self):
 
         try:
             employee_group = self.env.ref('base.group_user')
-            hr_suggestions = [{'id': user.partner_id.id, 'name': user.name, 'email': user.email} for user in employee_group.users]
+            hr_suggestions = [{
+                'id': user.partner_id.id,
+                'name': user.name,
+                'email': user.email,
+                'employee_id': user.employee_id.id if user.employee_id else None,
```

Comment:
Do we need specifically `None`? Because if we just leave `user.employee_id.id` it should be `False` if the relation doesn't exist.

Reply:
No, indeed